src/pudl/etl.py

```diff
@@ -234,9 +234,6 @@ def _etl_eia(etl_params, datapkg_dir, pudl_settings, ds_kwargs):
     eia_transformed_dfs = eia860_transformed_dfs.copy()
     eia_transformed_dfs.update(eia923_transformed_dfs.copy())
 
-    # Add EIA-EPA crosswalk tables
-    eia_transformed_dfs = _add_eia_epacems_crosswalk(eia_transformed_dfs)
```

Comment:
It's glue that connects the epacems and eia tables, and we have a section for that kind of thing (where we do the FERC to EIA glue) and so I wanted to put it there for organizational purposes. Austen put it off by itself, and then I incorrectly put it in the static table loading because it doesn't depend on anything else. I think the glue is where it really belongs, so long as that's a section we've got.